modules/http-helper/dummy_server.go

```diff
@@ -35,11 +35,17 @@ func RunDummyServerE(t *testing.T, text string) (net.Listener, int, error) {
 	logger.Logf(t, "Starting dummy HTTP server in port %d that will return the text '%s'", port, text)
 
 	listener, err := net.Listen("tcp", ":"+strconv.Itoa(port))
-
-	if err == nil {
-		go http.Serve(listener, nil)
+	if err != nil {
+		return nil, 0, fmt.Errorf("error listening: %s", err)
 	}
 
+	go func() {
+		err = http.Serve(listener, nil)
+		if err != nil {
+			panic(err)
```

Comment:
Again, we should generally not `panic` anywhere in Terratest... Not sure what effect a `panic` would have in a `goroutine` though... Would it result in the tests all exiting? Or just be ignored?

Reply:
I removed it (see comments above).